app/assets/javascripts/statement_of_case_file_upload.es6

```diff
@@ -0,0 +1,84 @@
+$(document).ready(() => {
+  if ($('input#original_files').length) {
+
+    const file_input = $('#original_files')
+    const form = file_input.parents('form')
+    const error_summary_container = $('#files-errors-container')
+    const uploaded_files_table_container = $('#uploaded-files-table-container')
+    const authenticity_token = $("[name=authenticity_token]").val()
+    const image_loading_small = window.LAA_VARS.images.loading_small
+    const copy_uploading = window.LAA_VARS.locales.generic.Uploading
+
+    function send_files(files, callback) {
+      const form_data = new FormData()
+
+      form_data.append('authenticity_token', authenticity_token)
+      Array.from(files).forEach(file => form_data.append('statement_of_case[original_files][]', file))
+
+      $.ajax({
+        url: form.attr('action'),
+        data: form_data,
+        processData: false,
+        contentType: false,
+        type: form.find('[name=_method]').val().toUpperCase(),
+        success: callback
+      })
+    }
+
+    function show_uploading_files(files) {
+      const rows = Array.from(files).map(file => (
+        `
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">${file.name}</td>
+            <td class="govuk-table__cell"></td>
+            <td class="govuk-table__cell no-wrap">
+              <img src="${image_loading_small}" class="small-loading-image"/>
+              ${copy_uploading}
+            </td>
+            <td class="govuk-table__cell"></td>
+          </div>
+        `
+      ))
+      uploaded_files_table_container.find('table').show().find('tbody').append(rows.join(''))
```

Comment:
Would this work:
```javascript
uploaded_files_table_container.find('table tbody').append(rows.join(''))
```

Reply:
We also need to call `show` on the table. When the statement of case has no file yet, the table is in the page but hidden. That way, when we add the first file, we just need to show the table and a row